include/openenclave/host.h

```diff
@@ -122,6 +123,17 @@ typedef struct _oe_enclave_setting_context_switchless
     size_t max_enclave_workers;
 } oe_enclave_setting_context_switchless_t;
 
+
+/**
+ * The setting for config_id/config_svn on Ice Lake platform.
+ */
+typedef struct _oe_enclave_setting_config_data
```

Comment:
Is this struct intended to be used with OE_SGX_ENCLAVE_CONFIG_ID as the setting type?  If so, the terms should match (i.e., SGX in the name, data vs id).   I see comment line 128 does imply SGX only so please rename this struct to be consistent with the setting type value.